Cannot find module '/node_modules/ISOToLanguage/lib/esm/index.js'

Very nice project, thank you!  ❤️ 
I noticed 2 things, though:

1. "Weighing less than 10kb gzipped" seems incorrect. Everywhere it says 2mb.
2. I can't import it:

```haskell
Error: Cannot find module '/services/whatsapp-listener/node_modules/ISOToLanguage/lib/esm/index.js' imported from /services/whatsapp-listener/context.ts
    at finalizeResolution (/node_modules/.pnpm/ts-node@10.9.2_@types+node@20.11.24/node_modules/ts-node/dist-raw/node-internal-modules-esm-resolve.js:366:11)
    at moduleResolve (/node_modules/.pnpm/ts-node@10.9.2_@types+node@20.11.24/node_modules/ts-node/dist-raw/node-internal-modules-esm-resolve.js:801:10)
    at Object.defaultResolve (/node_modules/.pnpm/ts-node@10.9.2_@types+node@20.11.24/node_modules/ts-node/dist-raw/node-internal-modules-esm-resolve.js:912:11)
    at /node_modules/.pnpm/ts-node@10.9.2_@types+node@20.11.24/node_modules/ts-node/src/esm.ts:218:35
    at entrypointFallback (/node_modules/.pnpm/ts-node@10.9.2_@types+node@20.11.24/node_modules/ts-node/src/esm.ts:168:34)
    at /node_modules/.pnpm/ts-node@10.9.2_@types+node@20.11.24/node_modules/ts-node/src/esm.ts:217:14
    at addShortCircuitFlag (/node_modules/.pnpm/ts-node@10.9.2_@types+node@20.11.24/node_modules/ts-node/src/esm.ts:409:21)
    at resolve (/node_modules/.pnpm/ts-node@10.9.2_@types+node@20.11.24/node_modules/ts-node/src/esm.ts:197:12)
    at nextResolve (node:internal/modules/esm/hooks:865:28)
    at Hooks.resolve (node:internal/modules/esm/hooks:303:30)
    ```

--- a/package.json
+++ b/package.json
@@ -27,7 +27,7 @@
         "types": "./lib/cjs/@types/index.d.ts"
       },
       "import": {
-        "default": "./lib/esm/index.js",
+        "default": "./lib/esm/index.mjs",
         "types": "./lib/@types/index.d.ts"
       },
       "browser": "./lib/browser/ISOToLanguage.js",

--- a/package.json
+++ b/package.json
@@ -23,7 +23,7 @@
     ".": {
       "types": "./lib/cjs/@types/index.d.ts",
       "require": {
-        "default": "./lib/cjs/index.js",
+        "default": "./lib/cjs/index.cjs",
         "types": "./lib/cjs/@types/index.d.ts"
       },
       "import": {

